Pipfile

```diff
@@ -18,6 +18,7 @@ pytest = "==6.2.4"
 pytest-cov = "==2.11.1"
 pytest-asyncio = "==0.14.0"
 pytest-xdist = "==2.2.1"
+s3fs = "==2023.3.0"
```

Comment:
this introduces a circular dependency, can't do it as it may later be incompatible with the version of aiobotocore this becomes

Reply:
Makes sense. Used `paginator` for testing